src/test/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantExportBatchControllerTest.java

```diff
@@ -97,4 +106,46 @@ void errorThrown() throws Exception {
 
     }
 
+    @Nested
+    class getExportBatchInfo {
+
+        @Test
+        void successfullyGetsBatchInfo() throws Exception {
+            UUID randomUUID = UUID.randomUUID();
```

Comment:
You could remove this and use `mockExportId` instead that's initialised at the top just to avoid duplication